src/pages/SettingsPage/UsersSettings/UserAttribForm.jsx

```diff
@@ -22,6 +22,7 @@ const UserAttribForm = ({
   setPasswordConfirm,
   setPassword,
   disabled,
+  showAvatarUrl = true,
```

Comment:
Small thing and feel free to ignore me. I always feel like booleans that have a default value of `true` should be inverted. For example `showAvatarUrl= true` goes to `hideAvatarUrl=false`. If showing the avatar is the default behaviour then the prop name should reflect the non-default behavior. What do you think?

Reply:
Not sure about this one. As far as i know, you should avoid "negative" names of boolean props, since they can get hard to read. 

That being said, i see your point and it is question wheter `hideAvatarUrl` is negative or not. 

This is what am i coming from, mazbe
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
<img width="580" alt="image" src="https://github.com/ynput/ayon-frontend/assets/16327908/f0494fea-f232-47b9-bdfa-7ea7fea6fcd3">

Reply:
Oh interesting! I like the references behind your thinking, unlike my totally un-researched opinion... fair enough lets stick with `showAvatarUrl`. Good to know!